front_end/lib/ui/admin_home_page/admin_home_page_provider.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:front_end/ui/admin_home_page/admin_home_page_bloc.dart';
+import 'package:front_end/ui/admin_home_page/admin_home_page_view.dart';
+
+class AdminHomeScreenProvider extends StatelessWidget {
```

Comment:
dont user statelsess widgets for providers.

![image](https://github.com/fcodelabs/training-ramp-up/assets/47570687/5b83099f-0afb-455e-8b70-e364af3eac31)

create like this.